<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WTC-5159473-l2" ignored="1" name="Default test" tests="2" failures="2" timestamp="18 Sep 2019 16:45:47 GMT" time="7.928" errors="0">
  <testcase name="Create_Shipment" time="1.085" classname="SHPC_Application.SHPC_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;b77cb968-b280-4f0e-989c-977263c8b919&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;SHIPMENT.PAYMENTYPE.INVALID&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Invalid payment type selected.  Please select a different payment method.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{"transactionId":"b77cb968-b280-4f0e-989c-977263c8b919","errors":[{"code":"SHIPMENT.PAYMENTYPE.INVALID","message":"Invalid payment type selected.  Please select a different payment method."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_Endpoints.Create_Shipment(SHPC_Endpoints.java:129)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="Create_Shipment" time="2.119" classname="SHPC_Application.SHPC_Endpoints">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;masterTrackingNumber&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;c1abfcff-9c10-44cc-a1f8-fb34e0ca5f35&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;SHIPMENT.PAYMENTYPE.INVALID&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Invalid payment type selected.  Please select a different payment method.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "masterTrackingNumber"
     but: was "{"transactionId":"c1abfcff-9c10-44cc-a1f8-fb34e0ca5f35","errors":[{"code":"SHIPMENT.PAYMENTYPE.INVALID","message":"Invalid payment type selected.  Please select a different payment method."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at SHPC_Application.SHPC_Endpoints.Create_Shipment(SHPC_Endpoints.java:129)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- Create_Shipment -->
  <testcase name="CreditCard_Linked" time="0.0" classname="SHPC_Application.SHPC_Endpoints">
    <ignored/>
  </testcase> <!-- CreditCard_Linked -->
</testsuite> <!-- Default test -->
